corehq/apps/translations/generators.py

```diff
@@ -1,16 +1,18 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
 
 import datetime
 import os
 import re
 import tempfile
-from collections import namedtuple, OrderedDict, defaultdict
+from collections import OrderedDict, defaultdict, namedtuple
 
-import polib
 from django.utils.functional import cached_property
+
+import polib
+import six
```

Comment:
should these two be on the top?

Reply:
Imports have been organised according to our [isort](https://isort.readthedocs.io/en/stable/) configuration, which you can find in `.isort.cfg`. It was introduced in PR #23863 so that all devs use identical settings. One of the benefits of using the same configuration is that it minimises merge conflicts in imports. I would strongly recommend you add it to your editor/IDE. It will make your life quantifiably better.

Reply:
cool, thanks for pointing that Norman. I thought global imports always go on top.

Reply:
Yes! I used to believe that too :wink: ... In my old age I have become more a fan of not having merge conflicts than of holding onto beliefs that, it has _sometimes_ turned out, were just chosen arbitrarily by a slightly different bunch of people. :thinking: :speak_no_evil: :upside_down_face: :joy: #lifeLesson

Reply:
haha.